### PR TITLE
Add general site styling, add template to directory page.

### PR DESCRIPTION
**Pull request by ganewman**
### What is a quick description of the change?

- Modify HTML and CSS in order to add general styling to the page. This involves adding fonts, background colors, margins/padding, and defining the positioning and styling on the navigation bar (though dynamic features will be added in the next PR).
- Rewrite the directory page template and corresponding component in order to utilize template-ing, as in on the candidate page. This involves the deletion of the script.js file as all code has now been moved into relevant components.

### Is this fixing an issue?

none

### Are there more details that are relevant?

none

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [X] Tests passing
- [X] Coding style (indentation, etc)

Please explain why any are not present, if any.

### Any additional comments?

### CODE REVIEW

File: project/src/main/webapp/directory-page/directory-page.component.js [hunk 1]
```diff
@@ -17,9 +17,26 @@
...
+  const listAllElections =
+      location.search.substring(location.search.lastIndexOf('=') + 1);
+
+  // Send GET request to /data with address and whether to list all elections.
+  const response = await fetch(`/data?address=${address}&listAllElections=${listAllElections}`);
+  const dataPackage = await response.json();
+
+  scope.elections = dataPackage.electionsData;
+  scope.$apply();
+}
```
Comment (suparnabarua) [inline project/src/main/webapp/directory-page/directory-page.component.js]: nit new line

File: project/src/main/webapp/navigation/nav-bar.template.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      <a class="nav-link" href="/index.html">Home</a>
+    </li>
+    <li class="navbar-item">
+      <a class="nav-link" href="/education.html">Voter Education</a>
+    </li>
+    <li class="navbar-item">
+      <a class="nav-link" href="/">Polling Locations</a>
+    </li>
+  </ul>
+</nav>
```
Comment (suparnabarua) [inline project/src/main/webapp/navigation/nav-bar.template.html]: nit. new line

File: project/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,4 +1,34 @@
...
+#navigation-bar a {
+  color: #474747;
+  font-family: 'Roboto Slab', serif;
+}
+
+#site-header {
+  color: #f0eded;
+  background-color: #b32424;
+  font-family: 'Roboto Slab', serif;
+}
```
Comment (suparnabarua) [inline project/src/main/webapp/style.css:34]: nit new line

File: project/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,4 +1,34 @@
...
+
+#navigation-bar {
+  background-color: #4788c4;
+  float: left;
+  font-size: 22px;
+  height: 100%;
+  position: fixed;
+}
+
+#navigation-bar a {
```
Comment (suparnabarua) [inline project/src/main/webapp/style.css]: Can you use a different name for the second navigation-bar?

**Review by suparnabarua — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/style.css (hunk 4 above)
Comment (ganewman) [reply project/src/main/webapp/style.css]: This rule styles the links (the `<a>` elements) within the sidebar. Is that what you're referring to?

File: project/src/main/webapp/directory-page/directory-page.component.js [hunk 5]
```diff
@@ -17,9 +17,26 @@
 
 angular.module('directoryPage').component('directoryPage', {
   templateUrl: '/directory-page/directory-page.template.html', 
-  controller: function directoryPageController() {
-    // TODO: modify this code so it works within the component, and interacts
-    // with the template rather than as a seperate JS function.
-      addBriefElectionCandidateInformation()
+  controller: function directoryPageController($scope) {
+      getElectionCandidateInformation($scope);
```
Comment (KathyFeiyang) [inline project/src/main/webapp/directory-page/directory-page.component.js]: nit: two spaces for indentation

File: project/src/main/webapp/navigation/header.component.js [hunk 6]
```diff
@@ -16,7 +16,9 @@
 "use strict"
 
 angular.module('navigation').component('header', {
-  template: `<h1>Voter Central</h1>`,
+  template: `<div class="py-2 mb-3" id="site-header">
+             <h1>Voter Central</h1>
```
Comment (KathyFeiyang) [inline project/src/main/webapp/navigation/header.component.js]: nit: indent <h1>...</h1>

File: project/src/main/webapp/style.css [hunk 7]
```diff
@@ -1,4 +1,34 @@
...
+  color: #474747;
+  font-family: 'Roboto Slab', serif;
+}
+
+#site-header {
+  color: #f0eded;
+  background-color: #b32424;
```
Comment (KathyFeiyang) [inline project/src/main/webapp/style.css]: extra small nit: use alphabetic order

**Review by KathyFeiyang — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/navigation/header.component.js [hunk 8]
```diff
@@ -17,7 +17,7 @@
 
 angular.module('navigation').component('header', {
   template: `<div class="py-2 mb-3" id="site-header">
-             <h1>Voter Central</h1>
+              <h1>Voter Central</h1>
```
Comment (KathyFeiyang) [inline project/src/main/webapp/navigation/header.component.js:20]: extra extra small nit: 2 indentations relative to div

**Review by KathyFeiyang — APPROVED** [review]
LGTM! Thanks for making these changes

**Review by suparnabarua — APPROVED** [review]

**Review by kellyktan — APPROVED** [review]
LGTM pending open comments

**Review by tjyoung99 — APPROVED** [review]

**Pull request merged**